src/localization.js

```diff
@@ -89,7 +91,17 @@ export default class Localization extends I18n {
     }
 
     translateString (string) {
-        return this.i18n(string)
+        const translated = this.i18n(string)
```

Comment:
This sometimes fails to engage the translation, it needs a fallback, which is done below. If the translation fails and simply returns a string property, it is compared against the translation file to get the translated correct string.